### ISSUE TITLE
Add 'Fetch' command to nudb tool

### ISSUE THREAD

**Issue by MarkusTeufelberger** [vinniefalco/NuDB#38]
Might be useful for external tools/scripts (e.g. inserting data, retreiving data and asserting that it is the same).

**Comment by vinniefalco** [comment]
This is great, but it would be even better as a pull request!

**Comment by MarkusTeufelberger** [comment]
I can try, but I'm neither very deep into modern C++ nor boost at the moment, so don't expect anything great...

**Comment by vinniefalco** [comment]
No worries, lets just design what such a feature might look like. How about
```
nudb fetch <dat-file> <key-file> <key>
```

Where `<key>` is a hexadecimal string?

**Comment by MarkusTeufelberger** [comment]
Yeah, that's about what I would be expecting. The command would then print the data (probably also in Hex) or an error message.
